Gemfile

```diff
@@ -1,5 +1,6 @@
 source 'https://rubygems.org'
 
+gem 'colorize', '0.8.1'
```

Comment:
Is there a point of this gem now that the new failure message doesn't show color?